desktop/NodeManager.ts

```diff
@@ -329,23 +329,45 @@ class NodeManager extends AbstractManager {
           isLocalStateFarBehind(qsStatus, 100)
         ) {
           await this.runQuicksyncPrompt(qsStatus);
+        } else if (
+          qsStatus.partial &&
+          qsStatus.db >= qsStatus.partial.from &&
+          qsStatus.partial.to >= qsStatus.current - 10
+        ) {
+          hideGenericModal(this.mainWindow.webContents);
+          const isForce = await showGenericPrompt(this.mainWindow.webContents, {
+            title: 'Quick sync is not needed',
+            message: [
+              `Partial quicksync available from layer ${qsStatus.partial.from} to ${qsStatus.partial.to}`,
+              `Latest layer in your database: ${qsStatus.db}`,
+              `Current layer in the network: ${qsStatus.current}`,
+              '',
+              'Your node is nearly synced. We recommend you to keep using your local state.',
+              'Do you want to partially update to the trusted state anyway?',
+            ].join('\n'),
+            confirmTitle: 'Update partially!',
```

Comment:
That one, too, Update partially, has a bad meaning for me. It suggests that the update is not full, etc.